### ISSUE TITLE
Implement post photos to the feed

### ISSUE THREAD

**Issue by LordArma** [stanvanrooy/instauto#121]
Hi. Thank you because of your great module. I use your module in my project. Everything is good but I can't see any example to send more than one picture to the feed.

I want to send photos as an gallery. Can you help by add a new example for this or implement this feature if isn't exist right now?

Thank you.

**Comment by stanvanrooy** [comment]
This currently is not supported, but it shouldn't be that much more work to implement. I'll put it on the to-do list.

**Comment by stanvanrooy** [comment]
See https://github.com/stanvanrooy/instauto/blob/master/examples/post/upload_carousel_to_feed.py and https://github.com/stanvanrooy/instauto/blob/master/examples/post/upload_carousel_with_usertags_to_feed.py

**Comment by LordArma** [comment]
> See https://github.com/stanvanrooy/instauto/blob/master/examples/post/upload_carousel_to_feed.py and https://github.com/stanvanrooy/instauto/blob/master/examples/post/upload_carousel_with_usertags_to_feed.py

Thank you very much. I have a question, what does caption meaning for each PostFeed? Normally we have a post with only one caption. What are these captions? Are they the alt tag of each image?

**Comment by stanvanrooy** [comment]
The example may be a bit confusing haha. I just copied it from another example, but the objects don't need a `caption` param. It isn't used.

The `post_carousel` method has an parameter `caption` for the caption.

**Comment by LordArma** [comment]
> The example may be a bit confusing haha. I just copied it from another example, but the objects don't need a `caption` param. It isn't used.
> 
> The `post_carousel` method has an parameter `caption` for the caption.

I tested this new feature and unfortunately got some errors. When I wanna use post_carousel without usertags I get this error:
"AttributeError: 'NoneType' object has no attribute 'to_dict'"
This happens in there:
```
            if hasattr(post, 'usertags'):
                usertags = post.usertags.to_dict()
                for i, usertag in enumerate(data['in']):
                    usertags['in'][i] = usertag.to_dict()
                data['children_metadata'][-1]['usertags'] = json.dumps(usertags)
```
Also when I wanna use post_carousel as your example I get another one:
"for i, usertag in enumerate(data['in']):
KeyError: 'in'"

Also I found that your pytest failed:
https://github.com/stanvanrooy/instauto/runs/1532349139

May you help me?

**Comment by stanvanrooy** [comment]
It wasn't working, because the `upload_id` was based upon the current timestamp(seconds).

The posts were being uploaded within the timespan of 1 second, so they ended up with the same upload id.

**Comment by LordArma** [comment]
> It wasn't working, because the `upload_id` was based upon the current timestamp(seconds).
> 
> The posts were being uploaded within the timespan of 1 second, so they ended up with the same upload id.

Thank you so much. When wll you update PyPi? It seems it's still on version 1.0.4.

**Comment by stanvanrooy** [comment]
@LordArma latest version is available on PyPi now :) 1.0.5

**Comment by stanvanrooy** [comment]
lmk if you run into any more issues.
